ShoppingList/ShoppingList/ShoppingItem.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ShoppingList
+{
+    public class ShoppingItem
+    {
+
+        public ShoppingItem(string v1)
```

Comment:
```suggestion
        public ShoppingItem(string name)
```
Unclear what v1 means.

Reply:
It was an auto generated field I forgot to change. Good find!

Reply:
Yeah, I figured! Visual Studio is very good at naming things haha. :)